tmt/utils/jira.py

```diff
@@ -196,7 +201,12 @@ def link(
     """
 
     # TODO: Shall we cover all relations instead?
-    for link in links.get("verifies"):
+    # The default relation is `relates`, we need to include it to capture
+    # links user provided without relation.
+    link_list: list[tmt.base.Link] = []
+    for relation in tmt.base.Links._relations:
+        link_list += links.get(relation)
+    for link in link_list:
```

Comment:
```suggestion
    for link in links.get():
```

What about just simply interating through all available links?

Reply:
ah, much better, updated :) https://github.com/teemtee/tmt/pull/3395/commits/2132f9a73f6f8ee0ea9ac735a9f7c09de39bc63c